addons/mail/static/src/models/message/message.js

```diff
@@ -111,8 +111,10 @@ function factory(dependencies) {
                     data2.parentMessage = insertAndReplace(this.convertData(data.parentMessage));
                 }
             }
-            if ('partner_ids' in data && this.messaging.currentPartner) {
-                data2.isCurrentPartnerMentioned = data.partner_ids.includes(this.messaging.currentPartner.id);
+            if ('partners_data' in  data) {
+                data2.recipients = insert(data.partners_data.map(partnerData =>
```

Comment:
Actually the example I gave you was not ideal, using `replace` would be better than using `insert` here (if a recipient is no longer present after edit, it should be removed).